db-client-java/src/main/java/com/eventstore/dbclient/ServerFeatures.java

```diff
@@ -44,7 +44,7 @@ private static CompletableFuture<ServerInfo> getSupportedFeaturesInternal(Server
         stub.getSupportedMethods(Shared.Empty.getDefaultInstance(), convertSingleResponse(result, resp -> {
             int major = 0, minor = 0, patch = 0;
 
-            String[] splits = resp.getEventStoreServerVersion().split(".");
+            String[] splits = resp.getEventStoreServerVersion().split("\\.");
```

Comment:
It's currently working, why does it need to be changed?

Reply:
based on my tests, it appears that the version is not parsed correctly (split() takes in a regex, not a delimiter)

Reply:
But if it were the case, it would mean it never worked before.

Reply:
yes, it appears to be the case but it wasn't noticeable since the version wasn't being used anywhere yet.

Reply:
e.g "22.6.0".split(".") returns a 0-length array, so major, minor and patch were all set to 0.

Reply:
I see, thanks for the explanation.